### DISCUSSION TITLE
Project enhancements: Add locality or something spatial

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4883] [General]
We can add Taxonomy to a project. How about locality? Working on [this project](http://arctos.database.museum/project/10003185), I got to thinking that it would be cool to add locality(ies) and I can do that with links in the project text, but how about creating links like we have with Taxonomy? Make sense?

Holy Cow! Look at that link!

![image](https://user-images.githubusercontent.com/5725767/61996696-5ddcbe80-b054-11e9-8ee5-655d690ca2f2.png)

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
This would be useful for us.
